app/controllers/cloud_volume_controller.rb

```diff
@@ -185,7 +185,6 @@ def new
     assert_privileges("cloud_volume_new")
     assert_privileges("cloud_tenant_show_list")
 
-    @volume = CloudVolume.new
```

Comment:
Please also modify haml:
https://github.com/ManageIQ/manageiq-ui-classic/blob/9ad557b14dd91cd151c51308429303aebd092849/app/views/cloud_volume/_common_new_edit.html.haml#L10
Here ^ it crashes with `undefined method 'id' for nil:NilClass`. We should probably test if `@volume` itself is nil, not try to access its id.

Reply:
Right. I fixed it and added a simple spec.